WordPress/Classes/Utility/PushAuthenticationManager.swift

```diff
@@ -103,20 +103,19 @@ import WordPressShared
     ///
     /// - Parameter userInfo: Is the Notification's payload.
     ///
-    fileprivate func isNotificationExpired(_ userInfo: NSDictionary?) -> Bool {
-        let rawExpiration = userInfo?["expires"] as? TimeInterval
-        if rawExpiration == nil {
+    func isNotificationExpired(_ userInfo: NSDictionary?) -> Bool {
```

Comment:
_Super_ nitpicky, but above we have `isPushAuthenticationNotification` and here we have `isNotificationExpired`... can we make them consistent and perhaps remove `Push` from the first one? (the class has Push in the name anyway)